rewrite-maven/src/main/java/org/openrewrite/maven/OrderPomElements.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.maven;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.maven.tree.Maven;
+import org.openrewrite.xml.AutoFormat;
+import org.openrewrite.xml.XmlVisitor;
+import org.openrewrite.xml.tree.Content;
+import org.openrewrite.xml.tree.Xml;
+
+import java.util.*;
+
+public class OrderPomElements extends Recipe {
```

Comment:
Can you please tag this with the proper sonar ID? Also you should have some docs that reference where the order was defined.

Reply:
oh, I see you reference that in the description.

Reply:
but we do need to add a tag, like we do for other sonar-based rules.